File: dotNet/astenn/astenn-core/src/Org/Lestr/Astenn/Configuration/IConfiguration.cs
using System;
using System.Collections.Generic;
using System.Text;

using Org.Lestr.Astenn.Plugin;

namespace Org.Lestr.Astenn.Configuration
{


    public interface IConfiguration
    {


        CompositePersistenceDriver PersistenceDriver { get; }


        IPermissionsManager PermissionsManager { get; }


    }// END Interface IConfiguration


}// END Interface Org.Lestr.Astenn.Configuration

File: astenn-core/src/main/java/org/lestr/astenn/configuration/IConfiguration.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package org.lestr.astenn.configuration;

import java.util.Map;

/**
 *
 * @author PIBONNIN
 */
public interface IConfiguration {


    public String getAstennInstanceId();


    public CompositePersistenceDriver getPersistenceDriver();


    public IPermissionsManager getPermissionsManager();


    public Map<String, Object> getProperties();


    public Map<String, Object> getCurrentThreadSpecificsProperties();


}// END Interface IConfiguration


